src/strings/unicode_ops.c

```diff
@@ -400,3 +412,35 @@ void MVM_unicode_release(MVMThreadContext *tc)
     }
     uv_mutex_unlock(&property_hash_count_mutex);
 }
+/* Looks up a grapheme by name. Lazily constructs a hash. */
+MVMString * MVM_unicode_string_from_name(MVMThreadContext *tc, MVMString *name) {
+    MVMuint64 size;
+    char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
+    MVMUnicodeGraphemeNameRegistry *result;
+
+    MVMGrapheme32 result_graph;
+    const MVMint32 * uni_seq;
+    int array_size;
+    MVMint32 cp;
+    MVMNormalizer norm;
+
+    result_graph = MVM_unicode_lookup_by_name(tc, name);
+    /* If it's just a codepoint, return that */
+    if (result_graph >= 0)
+        return MVM_string_from_grapheme(tc, result_graph);
```

Comment:
We could eliminate `MVM_string_from_grapheme` and just use `MVM_string_chr` here - which will take care of singleton decomposition, which this doesn't.